src/route.c

```diff
@@ -63,7 +63,7 @@ void route(meshlink_handle_t *mesh, node_t *source, vpn_packet_t *packet) {
 
 		char hex[len * 2 + 1];
 
-		if(mesh->log_level >= MESHLINK_DEBUG) {
```

Comment:
The value of MESHLINK_DEBUG is 0 but here the comment suggests that it's expected to be logged at DEBUG sort of scenario, What's the condition that we're expecting here? Also `mesh->log_level >= MESHLINK_DEBUG` statement will always be `true` which is a warning too.

Reply:
Ah, it should have been `mesh->log_level <= MESHLINK_DEBUG`.